content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods.rst

```diff
@@ -0,0 +1,103 @@
+Configuration
+=============
+
+Reservation methods are set on individual operations types. To configure reservation methods, go to
+:menuselection:`Inventory app --> Configuration --> Operations Types`. Then, select the desired
+operation type. Or, create a new one by clicking :guilabel:`New`.
+
+In the :guilabel:`General` tab of the operation type form, locate the :guilabel:`Reservation Method`
+option, and choose which method should be used for this type of operation.
+
+.. image:: reservation_methods/reservation-methods-operations-type-field.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+.. tip::
+   If the :guilabel:`Before scheduled date` reservation method is selected, a new
+   :guilabel:`Reserve before scheduled date` field appears below. From this field, the number of
+   :guilabel:`days before` and :guilabel:`days before when starred` can be changed from the default
+   `0`.
+
+   Changing the :guilabel:`days before` value changes the maximum number of days before a scheduled
+   date that products should be reserved.
+
+   Changing the :guilabel:`days before when starred` value changes the maximum number of days before
+   a scheduled date that starred (favorited) transfers for products should be reserved.
+
+   .. image:: reservation_methods/reservation-methods-before-scheduled-date.png
+      :align: center
+      :alt: Reserve before scheduled date fields with before scheduled date method chosen.
+
+Required applications
+=====================
+
+The two required applications that **must** be installed to use reservation methods are the *Sales*
+and *Inventory* apps.
+
+To install these apps, from the main Odoo dashboard, navigate to the :menuselection:`Apps`
+application, and locate the *Sales* and *Inventory* apps. Then, click :guilabel:`Install` on each
+app, and refresh the database.
+
+.. note::
+   In addition to delivery orders, reservation methods can also be used for *manufacturing orders*,
+   *resupply subcontractor* orders, orders for *repairs*, and *internal transfers*. To enable this,
+   configure the additional settings:
+
+   - **For manufacturing orders:** Install the *Manufacturing* application by going to the
+     :menuselection:`Apps` application, locating the *Manufacturing* app, and clicking
+     :guilabel:`Install`.
+   - **For resupply subcontractor:** Navigate to :menuselection:`Manufacturing app --> Configuration
+     --> Settings`, and under the :guilabel:`Operations` section, enable :guilabel:`Subcontracting`.
+     Then, click :guilabel:`Save`.
+   - **For repairs:** Install the *Repairs* application by going to the :menuselection:`Apps`
+     application, locating the *Repairs* app, and clicking :guilabel:`Install`.
+   - **For internal transfers:** Navigate to :menuselection:`Inventory app --> Configuration -->
+     Settings`, and under the :guilabel:`Warehouse` section, enable :guilabel:`Storage Locations`.
+     Then, click :guilabel:`Save`.
```

Comment:
Wouldnt the _Manufacturing_ and _Repairs_ apps be **Required applications** in this case? If so, it would make sense to me to include these apps in the previous instructions to install the _Sales_ and _Inventory_ apps. 

Doing so would also simplify the bullet points to show just the config of **For resupply subcontractor** and **For internal transfers** instead of both installing apps and config.

I also may not be understanding this section correctly, so please let me know your thoughts. There isnt anything technically wrong here, just voicing my thoughts 🙂

Reply:
My logic with this section was that you don't actually need to use reservation methods for these functions if you don't want, since it's most commonly used on delivery orders. So the info in this list is just telling people how to go about setting up the "optional" extras, so to speak, if that makes sense.

Reply:
I added some words for clarification to try and make this a bit more clear